virt-v2v/pkg/customize/scripts/rhel/run/network_config_util.sh

```diff
@@ -133,9 +173,20 @@ udev_from_netplan() {
         return 0
     fi
 
+    # Function to check if netplan supports the 'get' subcommand
+    netplan_supports_get() {
+        netplan get 2>/dev/null
```

Comment:
Ah add it here as well

Reply:
changed

note, this log line is less interesting, because failing here is a "happy path", this method is for testing if this command will error or not, failing is one of the expected paths.

Reply:
yeah aware of it, just want to log in case we will see a problem with the happy path